parity/cli/mod.rs

```diff
@@ -318,6 +318,10 @@ usage! {
 			"--db-path=[PATH]",
 			"Specify the database directory path",
 
+			ARG arg_sync_until: (Option<u64>) = None, or |c: &Config| c.parity.as_ref()?.sync_until.clone(),
+			"--sync-until=[NUM]",
+			"Sync until the given block has been imported, then enter offline mode. Intended for debug/benchmarking only.",
```

Comment:
@adria0 my bad, I had done the changes a long time ago but never pushed. 🤦

Reply:
Thanks david! :)